samples/isilon_v230_k8s_125.yaml

```diff
@@ -0,0 +1,230 @@
+apiVersion: storage.dell.com/v1
```

Comment:
Why k8s 1.25 yaml added to 2.3.0 config version . Support for the same will be added from 2.5.0 config version.

Reply:
Removed the 1.2 version support

Reply:
changed